README.md

```diff
@@ -149,10 +149,13 @@ file of the accompanying Helm Chart.
 
 ### Install with Helm
 
-You can install the chart from the local filesystem.
+You can install the chart from the chart museum ( index file location: https://lightbend.github.io/kafka-lag-exporter/repo/index.yaml ).
```

Comment:
```suggestion
You can install the chart from the chart repository ( index file location: https://lightbend.github.io/kafka-lag-exporter/repo/index.yaml ).
```

Chart museum is a separate project, isn't it?

Reply:
right ; )